example_create_document_test.go

```diff
@@ -54,6 +54,7 @@ func Example_createDocument() {
 	if err != nil {
 		log.Fatalf("Failed to create database: %v", err)
 	}
+	defer db.Remove(nil)
```

Comment:
Please remove this change

Reply:
Ok, why is this test existing? It does not run more than one time.

Reply:
This is not really a test. It is example code which needs to have a name such as this to avoid inclusion in the normal code.

Reply:
That seems a bit odd to me. The tests - that I usually see in the `*_test` package - are placed in the `test` folder. And the examples - that I usually see in the `example` folder - are in the `*_test` package :-)

Anyway - that exceeds the scope of this PR and should not concern me.